src/build/__init__.py

```diff
@@ -59,6 +59,16 @@ class TypoWarning(Warning):
     """
 
 
+def _validate_source_directory(srcdir):
+    # type: (str) -> None
+    if not os.path.isdir(srcdir):
+        raise BuildException('Source {} is not a directory'.format(srcdir))
+    pyproject_toml = os.path.join(srcdir, "pyproject.toml")
+    setup_py = os.path.join(srcdir, "setup.py")
+    if not (os.path.exists(pyproject_toml) or os.path.exists(setup_py)):
```

Comment:
nitpick: I find it easier to read in the expanded form using De Morgan's laws. I do not need to keep in mind that a logical block is open and mentally negate the statement.

Reply:
Also fixed. Thanks!

Reply:
Thanks :blush: